index.html

```diff
@@ -20,79 +20,80 @@
       <nav id="navbar">
         <ul class="nav-links">
           <li><a href="#welcome-section" class="nav-link">Welcome</a></li>
-          <li><a href="#projects" class="nav-link">Projects</a></li>
+          <li><a href="#projects" class="nav-link">Projects</a>
           <li><a href="#contact" class="nav-link">Contact</a></li>
         </ul>
       </nav>
     </header>
    
     <!-- Welcome section -->
     <section id="welcome-section">
-      <h1>Hello, We are Moises & Cristian</h1>
+      <h4>Hello, I am Cristian</h4>
```

Comment:
ERROR: Should be <h1> instead of <h4>

Reply:
> ERROR: Should be h1 instead of h4

:red_circle: You have well asked for this change 👌. But I think this comment is not clear enough to the reader **Why** h1 should be used instead of h4 

You  could say: :arrow_down: 
- [ ] Great job so far! 👍 I kindly recommend using an <h1> tag instead of "h1". Using an <h1> tag instead of "h1" is a best practice in HTML coding. The <h1> tag is a semantic element specifically designed for defining the top-level heading on a webpage. It not only helps to structure your content but also provides accessibility benefits for screen readers and search engine optimization (SEO) purposes. By using the appropriate HTML tags, you ensure that your webpage is well-organized, accessible, and follows industry standards.

- Briefly for every section that needs an update try to point it out and tell the student what changes you expect from the section and why the change is needed.